website/docs/core-concepts/connecting-to-data-sources/authentication/README.md

```diff
@@ -1,63 +1,124 @@
 
-By default, Appsmith supplies a running sequence for the data source created. You can click on the pencil icon next to it to edit the name of the data source.
+```js
+https://mock-api.appsmith.com/users?page=1
+```
 
-:::info
-It is advisable that you give a meaningful name to your Authenticated API datasource.
-:::
+This is a mock API exposed by Appsmith to help you learn API basics. Data in the mock API have the following structure:
 
-### URL
+```js
+{
+  "next": "https://mock-api.appsmith.com/users?page=2&pageSize=10",
+  "previous": "https://mock-api.appsmith.com/users?page=0&pageSize=10",
+  "users": [
+    {
+      "id": 2,
+      "name": "Jenelle Kibbys",
+      "status": "APPROVED",
+      "gender": "Female",
+      "avatar": "https://robohash.org/quiaasperiorespariatur.bmp?size=100x100&set=set1",
+      "email": "jkibby1@hp.com",
+      "address": "85 Tennessee Plaza",
+      "createdAt": "2019-10-04T03:22:23.000Z",
+      "updatedAt": "2019-09-11T20:18:38.000Z"
+    },
+    {...
+```
 
-Use this field to add the API URL you would like to access. For example, I would like to access the mock API of Appsmith, and for that, I’ll supply the URL as `https://mock-api.appsmith.com`
+<VideoEmbed host="youtube" videoId="DWLF0pNjjuI" title="Using A Sample API " caption="How to use mock API | Example"/>
 
-### Headers
+ To use these mock APIs,
 
-You can add one or more header details that you would like to send along with the request to access the API you have integrated.
+* Click on **+** next to Datasources.
+* Select **REST API**
+* To ```fetch users``` choose the GET method and enter the following URL:
 
-### Query Parameters
+```js
+ GET 'https://mock-api.appsmith.com/users?page=1'
+```
 
-You can add one or more query parameters to the API you integrate with as part of the request.
 
-### Send Appsmith Signature Header
+The [API pane](/core-concepts/connecting-to-data-sources/authentication/connect-to-apis) is a REST interface that allows you to create and modify your existing APIs.
 
-When you want to ensure that the incoming requests originate from Appsmith, you can enable `Send Appsmith Signature Header` by selecting **Yes**. You’ll see a new field - **Session Details Signature Key** to supply the signature key.
 
-![Appsmith Signature Header](</img/OAuth__API_Integration__Appsmith_Signature_Header__Enable__.png>)
 
-### Authentication Type
+### Creating an authenticated API datasource
 
-You can define an [authentication type](/core-concepts/connecting-to-data-sources/authentication/authentication-type) for REST APIs by using the protocols available on Appsmith.
+When you create an authenticated API datasource, you need to configure the headers and query parameters only once and thereafter don't need to configure these for every API request. Further, to ensure authorized access and security, you can configure OAuth 2.0, signature headers and self-signed certificates for your API requests.
```

Comment:
Should we mention that there are other authentication mech supported as well like bearer token based, api key based etc.

Reply:
updated